pac4j-cas/src/main/java/org/pac4j/cas/client/rest/HttpTGTProfile.java

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright 2012 - 2015 pac4j organization
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package org.pac4j.cas.client.rest;
+
+import org.pac4j.http.profile.HttpProfile;
+
+/**
+ * This is {@link HttpTGTProfile}.
+ *
+ * @author Misagh Moayyed
+ * @since 1.8.0
+ */
+public final class HttpTGTProfile extends HttpProfile {
```

Comment:
All profiles must support to be built using this method: https://github.com/pac4j/pac4j/blob/master/pac4j-core/src/main/java/org/pac4j/core/profile/ProfileHelper.java#L56 So you certainly need to add a few lines of code for this new profile.